packages/ses/test/error/throws-and-logs.js

```diff
@@ -19,11 +17,6 @@ const compareLogs = freeze((t, log, goldenLog) => {
   t.is(log.length, goldenLog.length, 'wrong log length');
   log.forEach((logRecord, i) => {
     const goldenRecord = goldenLog[i];
-    t.is(
-      logRecord.length,
-      goldenRecord.length,
-      `wrong length of log record ${i}`,
-    );
```

Comment:
This is uselessly redundant with the assertions later in the function, and makes the failure from `ava` less informative.

Reply:
Let’s instead move the length check after the individual record check. Removing this check entirely would hide a defect where the actual log is shorter than the expected log, since we’re iterating the actual log.

Reply:
Done in [cf6d268](https://github.com/endojs/endo/pull/1962/commits/cf6d2684000b216353e638b5dfb76e402f9cdd4f)